v3/public-resolvers.md

```diff
@@ -2177,9 +2177,25 @@ sdns://AQcAAAAAAAAAKFsyMDAxOjE5ZjA6NzQwMjoxNTc0OjU0MDA6MmZmOmZlNjY6MmNmZl0g7Uk9j
 ## wevpn-adblock-uswest
 
 Private DNSCrypt service by WeVPN in Los Angeles, USA - https://wevpn.com/dns
-With ad blocking.
+With ad, malware, phishing, cryptojacking blocking (DNS Firewall).
+
+sdns://AQcAAAAAAAAAEjcyLjExLjEzNC45MToxNTM1MyBV2R9ORyXQIRO7dlT0F2mYeEdvXEuBOWMdFkripZcN9B8yLmRuc2NyeXB0LWNlcnQuc2VjdXJlLmRucy50ZXN0
+
+
+## wevpn-adblock-useast
+
+Private DNSCrypt service by WeVPN in New Jersey, USA - https://wevpn.com/dns
+With ad, malware, phishing, cryptojacking blocking (DNS Firewall).
+
+sdns://AQcAAAAAAAAAFDIzLjIyNi4xMzQuMjQzOjE1MzUzIBf1guPReaoN_V7w-UelcO4YnvqoXnDGsxRPIRc6uSjsHzIuZG5zY3J5cHQtY2VydC5zZWN1cmUuZG5zLnRlc3Q
+
+
+## wevpn-adblock-uk
+
+Private DNSCrypt service by WeVPN in London, UK - https://wevpn.com/dns
+With ad, malware, phishing, cryptojacking blocking (DNS Firewall).
 
-sdns://AQMAAAAAAAAAEjcyLjExLjEzNC45MToxNTM1MyBV2R9ORyXQIRO7dlT0F2mYeEdvXEuBOWMdFkripZcN9B8yLmRuc2NyeXB0LWNlcnQuc2VjdXJlLmRucy50ZXN0
```

Comment:
That was used to work. Why change it?

Reply:
Just updating it to follow their sites addresses. If you visit wevpn.com/wedns then scroll down to "anon dnscrypt" then toggle dns firewall as "on" all of the sites (US West, US East, Europe and Asia) start with "sns://AQc" instead of the removed "sns://AQM"

Reply:
These are all wrong, then. The `No filter` flag shouldn't be set in their "firewall mode", since this is blocking queries.

Reply:
I do agree, however most of the ones that wereoriginally added (note, currently listed) are under the "no filter" one. This is why I left the original ones as the "wevpn-country" format then added the "wevpn-adblock-country" for people that do want to use the filtering. wevpn-adblock-useast used to exist as well, however it was renamed to wevpn-useast (not sure of the blocking status of it). While I did initially see some inconsistency, I chose to paste all (yes, even the broken asia/singapore one) into the list so that wevpn users had the options of any country as the previously listed ones didn't really block anything except the wevpn-adblock-uswest.